staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/sample_and_watermark.go

```diff
@@ -161,14 +162,15 @@ func (saw *sampleAndWaterMarkHistograms) SetX1(x1 float64) {
 
 func (saw *sampleAndWaterMarkHistograms) innerSet(updateXOrX1 func()) {
 	var when time.Time
+	whenP := &when
```

Comment:
probably worth a comment and a pointer to the go issue so this doesn't get removed prematurely by some well-meaning person who (correctly) identifies this code as unecessary

Reply:
Fixed in force-push to 9755d2ae416, which just adds a comment on the end of this line.